src/ext_slice.rs

```diff
@@ -587,12 +587,7 @@ pub trait ByteSlice: Sealed {
     #[cfg(feature = "std")]
     #[inline]
     fn repeatn(&self, n: usize) -> Vec<u8> {
-        let bs = self.as_bytes();
-        let mut dst = vec![0; bs.len() * n];
-        for i in 0..n {
```

Comment:
for an empty byte slice and n == usize::MAX, this loop will spin for a very long time for no reason.